Gemfile.lock

```diff
@@ -38,15 +38,13 @@ GEM
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
     addressable (2.4.0)
-    aws-sdk (1.66.0)
-      aws-sdk-v1 (= 1.66.0)
+    aws-sdk (1.50.0)
```

Comment:
I think you can probably leave this at `1.66.0` actually and just tweak the `gemspec` below. Thanks!